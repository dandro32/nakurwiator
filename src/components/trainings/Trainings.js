import React from 'react';
import Dashboard from '../shared/dashboard/Dashboard';

const Trainings = () => {
  const cards = [
    {
      title: 'Dodaj nowy trening',
      desc: 'Zaplanuj sobie nowy trening i sprawdź pompę',
      href: '/trainigns/new',
      hrefTitle: 'Dawej',
      imgSrc: '/img/training_new.jpg'
    },
    {
      title: 'Lista treningów',
      desc: 'Obczaj swoje treningi, edytuj lub usuwaj',
      href: '/trainigns/list',
      hrefTitle: 'Sprawdzam',
      imgSrc: '/img/training_list.jpg'
    }
  ];
  return (<Dashboard cards={cards} />);
};

export default Trainings;