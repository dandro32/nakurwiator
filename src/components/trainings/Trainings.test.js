import React from 'react';
import Trainings from './Trainings';
import TestRenderer from 'react-test-renderer';

jest.mock('../shared/dashboard/Dashboard', () => 'Dashboard component');

describe('Trainings component tests', ()=> {
  let result;

  beforeEach(() => {
    result = TestRenderer.create(
      <Trainings />
    ).toTree();

  });

  it('renders correctly', () => {
    expect(result).toMatchSnapshot();
  });
});