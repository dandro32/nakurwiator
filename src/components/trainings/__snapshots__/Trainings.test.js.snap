// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Trainings component tests renders correctly 1`] = `
Object {
  "instance": null,
  "nodeType": "component",
  "props": Object {},
  "rendered": Object {
    "instance": null,
    "nodeType": "host",
    "props": Object {
      "cards": Array [
        Object {
          "desc": "Zaplanuj sobie nowy trening i sprawdź pompę",
          "href": "/trainings/new",
          "hrefTitle": "Dawej",
          "imgSrc": "/img/training_new.jpg",
          "title": "Dodaj nowy trening",
        },
        Object {
          "desc": "Obczaj swoje treningi, edytuj lub usuwaj",
          "href": "/trainings/list",
          "hrefTitle": "Sprawdzam",
          "imgSrc": "/img/training_list.jpg",
          "title": "Lista treningów",
        },
      ],
    },
    "rendered": Array [],
    "type": "Dashboard component",
  },
  "type": [Function],
}
`;
